### ISSUE TITLE
[QA] Experimental VFS not available in branded mac builds

### ISSUE THREAD

**Issue by HanaGemela** [owncloud/docs-client-branding#53]
### Pre-submission Checks

- [X] I [checked for similar issues](https://github.com/owncloud/client/issues?q=), but could not find any. I also checked the closed issues. I could not contribute additional information to any existing issue.
- [X] I will take the time to fill in all the required fields. I know that the bug report may be dismissed otherwise due to lack of information.

### Describe the QA issue

Experimental VFS not available in branded mac builds
Client 3.1.0 https://cloud.owncloud.com/index.php/f/5962228

### Steps to reproduce the issue

1. Build a branded client with:
VFS - yes 
Enforce virtual files - yes

2. Check the availability of VFS

### Screenshots

No screenshots

### Expected behavior

Experimental feature is available

### Actual behavior

Experimental feature is not available

**Comment by michaelstingl** [comment]
Experimental features not available in branded builds (except testpilotcloud)

**Comment by HanaGemela** [comment]
How come testpilotcloud has it? What settings is it?

**Comment by michaelstingl** [comment]
> How come testpilotcloud has it? What settings is it?

@fmoc point to code?

**Comment by fmoc** [comment]
It's `bool enableExperimentalFeatures() const` in `theme.h`, see https://github.com/owncloud/client/blob/master/src/libsync/theme.h#L424-L430. This method's behavior cannot be configured from ownbrander. For `testpilotcloud`, the method has been overwritten to explicitly return `true`.

**Comment by michaelstingl** [comment]
Yes, experimental features should be never ever be enabled in branded builds, except testpilotcloud.
